README.md

```diff
@@ -91,34 +153,62 @@ ktlint {
 }
 ```
 
-## Samples
+### Samples
 
-Check [samples](samples/) folder that provides examples how-to apply plugin.
+This repository provides following examples how to setup this plugin:
+- [android-app](/samples/android-app) - applies plugin to android application project
+- [kotlin-gradle](/samples/kotlin-gradle) - applies plugin to plain Kotlin project that uses groovy in `build.gradle` files
+- [kotlin-js](/samples/kotlin-js) - applies plugin to kotlin js project
+- [kotlin-ks](/samples/kotlin-ks) - applies plugin to plain Kotlin project that uses Kotlin DSL in `build.gradle.kts` files
+- [kotlin-multiplatform-common](/samples/kotlin-multiplatform-common) - applies plugin to Kotlin common multiplatform module
+- [kotlin-multiplatform-js](/samples/kotlin-multiplatform-js) - applies plugin to Kotlin Javascript multiplatform module
+- [kotlin-multiplatform-jvm](/samples/kotlin-multiplatform-jvm) - applies plugin to Kotlin JVM multiplatform module
+- [kotlin-native](/samples/kotlin-native) - applies plugin to Kotlin native project
+- [kotlin-rulesets-using](/samples/kotlin-rulesets-using) - adds custom [example](/samples/kotlin-ruleset-creating) ruleset
 
 ## Tasks Added
 
+### Main tasks
+
 This plugin adds two tasks to every source set: `ktlint[source set name]Check` and `ktlint[source set name]Format`.
 Additionally, a simple `ktlintCheck` task has also been added that checks all of the source sets for that project.
 Similarly, a `ktlintFormat` task has been added that formats all of the source sets.
 
 If project has subprojects - plugin also adds two meta tasks `ktlintCheck` and `ktlintFormat` to the root project that
 triggers related tasks in subprojects.
 
-### Apply to IDEA
+### Additional helper tasks
+
 Two another tasks added:
 - `ktlintApplyToIdea` - Task generates IntelliJ IDEA (or Android Studio) Kotlin
                         style files in project `.idea/` folder.
 - `ktlintApplyToIdeaGlobally` - Task generates IntelliJ IDEA (or Android Studio) Kotlin
                                 style files in user home IDEA
                                 (or Android Studio) settings folder.
 
-They are always added only to the root project.
+They are always added **only** to the root project.
```

Comment:
👍 Would be good if we had a test that guaranteed this. Not something I expect you to do though.